File: Problem_of_the_Day/TwoSum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        indexes = {}
        for i, n in enumerate(nums):
                difference = target-nums[i]
                if difference in nums and i != nums.index(difference):
                    return i, nums.index(target-n)

File: Problem_of_the_Day/TwoSum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int n=nums.size();
        unordered_map<int,int> idx;
        for(int i=0;i<n;i++){
            int difference=target-nums[i];
            if(idx.find(difference)!=idx.end()){
                return {i,idx[difference]};
            }
            idx[nums[i]]=i;
        }
        return {};       
    }
};
